src/main.controller.js

```diff
@@ -884,13 +884,30 @@ plugin.controller('wgnConfigCtrl', ['$scope', '$q', '$routeParams', 'znData', 'z
 			var promise = $q.when(config);
 			var multiWebhooks = _webhook && Array.isArray(_webhook.options);
 
-			if (_webhook && !('webhookId' in config || 'webhook0Id' in config)) {
+			/**
```

Comment:
No need to include 'UPDATE:' in comments. Also, please use inline comments for comments within functions/methods: // comment and doc block style comments for function/method descriptions